README.md

```diff
@@ -2,6 +2,93 @@
 
 Build images from the commandline in a convenient way.
 
+## Installation
+
+This project is under development right now and needs to be run via:
+```
+$ go run github.com/osbuild/image-builder-cli/cmd/image-builder@main
+```
+we plan to provide rpm packages as well.
+
+
+## Prerequisites
+
+Make sure to have `osbuild` installed:
+```
+$ dnf install osbuild
+```
```

Comment:
Depending on what you're going to build, you might also need some subpackages (luks, lvm, ostree, ...).  More importantly though, you're gonna need ~a bigger boat~ `osbuild-depsolve-dnf`.